src/compas/geometry/frame.py

```diff
@@ -109,6 +109,14 @@ def __repr__(self):
             self.yaxis,
         )
 
+    def __str__(self):
+        return "{0}(point={1}, xaxis={2}, yaxis={3})".format(
+            type(self).__name__,
+            self.point.__str__(),
+            self.xaxis.__str__(),
+            self.yaxis.__str__(),
```

Comment:
I would call `str()` instead of invoking the dunder method directly:
```suggestion
            str(self.point),
            str(self.xaxis),
            str(self.yaxis),
```